dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/ConfigurableBrowse.java

```diff
@@ -792,6 +794,20 @@ else if (bi.isItemIndex() && !bi.isInternalIndex())
         return params;
     }
 
+    private void updateOffset(Request request, BrowseParams params) {
```

Comment:
Perhaps reformat this method to follow the DSpace [coding conventions](https://wiki.duraspace.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-CodingConventions).